output/src/main/scala/Incompat7.scala

```diff
@@ -13,6 +13,6 @@ object Incompat7 {
 
     implicit val writer: Writer[Map[List[Int], Int]] = ???
 
-    rw(Map.apply[scala.collection.immutable.List[Int], Int](List(1) -> 1))
```

Comment:
Why do we have those changes on `List`?

Reply:
I don't know. I was surprised too. I don't think that the global from ScalfixGlobal and the global from nsc.interactive are different, but they produced this change ...